tests/chainer_tests/functions_tests/array_tests/test_concat.py

```diff
@@ -83,22 +90,23 @@ def check_forward(self, inputs, backend_config):
     def test_forward(self, backend_config):
         self.check_forward(self.inputs, backend_config)
 
-    def check_backward(self, inputs, backend_config):
+    def check_backward(self, inputs, grad_outputs, backend_config):
         if backend_config.use_cuda:
             inputs = cuda.to_gpu(inputs)
-
-        inputs = [chainer.Variable(x) for x in inputs]
+            grad_outputs = cuda.to_gpu(grad_outputs)
 
         with backend_config:
-            y = functions.concat(inputs, axis=self.axis)
-            y.grad = y.data
-            y.backward()
-
-        for x in inputs:
-            testing.assert_allclose(x.data, x.grad, atol=0, rtol=0)
+            gradient_check.check_backward(
+                self._forward, inputs, grad_outputs,
+                **self.check_backward_options)
 
     def test_backward(self, backend_config):
-        self.check_backward(self.inputs, backend_config)
+        grad_outputs = [
+            numpy.random.uniform(
+                -1, 1, self.y_expected.shape
+            ).astype(self.dtype)
+        ]
```

Comment:
How about setting `grad_outputs` as a member attribute in `setUp` as you did below. Not a strong opinion but for consistency?

Reply:
Thanks for the quick fix!